pkg/kubelet/kubelet_test.go

```diff
@@ -293,9 +285,7 @@ func TestSyncLoopTimeUpdate(t *testing.T) {
 	kubelet := testKubelet.kubelet
 
 	loopTime1 := kubelet.LatestLoopEntryTime()
-	if !loopTime1.IsZero() {
-		t.Errorf("Unexpected sync loop time: %s, expected 0", loopTime1)
-	}
+	assert.True(t, loopTime1.IsZero(), "Sync loop time")
```

Comment:
can't really switch to `assert.Zero` since we need to call `IsZero', will add explanation.